common/lib/dependabot/file_parsers/base/dependency_set.rb

```diff
@@ -21,6 +21,7 @@ def initialize(dependencies = [], case_sensitive: false)
         attr_reader :dependencies
 
         def <<(dep)
+          return if dep.nil?
```

Comment:
Is this change required? Less sure about changing the common classes unless we need to and make use of the change across each package manager to not introduce subtle differences in implementation.

Reply:
> Is this change required?

Nope. I'll revert it.